scripts/imageSize.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable */
+const path = require('path');
+const fs = require('fs');
+
+let fileArray = [];
+
+const read = dir =>
```

Comment:
I know this is something small, but could this be renamed to `readDir`?  `read` feels vague to me.